app/tests/beaker-plugin-safe-app/test_modules/helpers.js

```diff
@@ -0,0 +1,115 @@
+  unregisteredUri: 'safe-dW5yZWdpc3RlcmVk:AQAAAMga9SYCAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=',
+  containersUri: 'safe-bmV0Lm1haWRzYWZlLnRlc3Qud2ViYXBwLmlk:AQAAALDJZuUBAAAAAAAAAA==',
+  sharedMdataUri: 'safe-bmV0Lm1haWRzYWZlLmV4YW1wbGVzLnRlc3QtYXBw:AQAAADvUUzgDAAAAAAAAAA=='
+};
+
+const appInfo2 = {
+  id: 'alternate.net.maidsafe.beaker_plugin_safe_app.test',
+  name: 'alternate_app',
+  vendor: 'alt MaidSafe Ltd.',
+  scope: null
+}
+
+const TAG_TYPE_DNS = 15001;
+const TAG_TYPE_WWW = 15002;
+
+//  Replaces the entire content of the file when writing data.
+const OPEN_MODE_OVERWRITE = 1;
+
+//  Appends to existing data in the file.
+const OPEN_MODE_APPEND = 2;
+
+//  Open file to read.
+const OPEN_MODE_READ = 4;
+
+//  Read the file from the beginning.
+const FILE_READ_FROM_BEGIN = 0;
+
+//  Read entire contents of a file.
+const FILE_READ_TO_END = 0;
+
+const permissions = {
+    _public: ['Read', 'Insert', 'Update', 'Delete', 'ManagePermissions'],
+    _publicNames: ['Read', 'Insert', 'Update', 'Delete', 'ManagePermissions']
+}
+
+const ownContainer =  { own_container: true };
+
+const initialiseApp = (customAppInfo, scope) => {
+  const fixed = {
+      id: 'net.maidsafe.beaker_plugin_safe_app.test',
+      name: 'beaker_plugin_safe_app_test',
+      vendor: 'MaidSafe Ltd.'
+  };
+
+  const appInfo = customAppInfo || fixed;
+
+  appInfo.scope = scope;
+  return window.safeApp.initialise(appInfo);
+}
+
+const authoriseApp = (appHandle) => {
+  return window.safeApp.authorise(appHandle, permissions, ownContainer);
+}
+
+const authoriseAndConnect = async () => {
+  let appHandle = await initialiseApp();
+  let authUri = await authoriseApp(appHandle);
+  return window.safeApp.connectAuthorised(appHandle, authUri);
+}
+
+const createNfsEmulation = async () => {
+  const mdHandle = await createRandomPublicMutableData();
+  return window.safeMutableData.emulateAs(mdHandle, 'NFS');
+}
+
+const createRandomPrivateMutableData = async () => {
+  const appHandle = await window.safeApp.initialise(appInfo2);
+  const authUri = await authoriseApp(appHandle);
+  await window.safeApp.connectAuthorised(appHandle, authUri);
+  const mdHandle = await window.safeMutableData.newRandomPrivate(appHandle, TAG_TYPE_DNS);
+  return window.safeMutableData.quickSetup(mdHandle, {});
+}
+
+const createRandomPublicMutableData = async (entriesObject) => {
+  const appHandle = await window.safeApp.initialise(appInfo2);
+  const authUri = await authoriseApp(appHandle);
+  await window.safeApp.connectAuthorised(appHandle, authUri);
+  const mdHandle = await window.safeMutableData.newRandomPublic(appHandle, TAG_TYPE_DNS);
+  return window.safeMutableData.quickSetup(mdHandle, entriesObject || {});
+}
+
+const createRandomXorName = () => crypto.randomBytes(32);
+
+const createUnownedMutableData = async () => {
```

Comment:
It might be simpler/cleaner (one less thing to miss), to just do: `export const createUnownedMutableData = ` and then you don't need to reexport at the bottom of the file.

Reply:
right, I just need to figure out the proper babelify configuration in `gulpfile` to transform `import/ export`, otherwise, as it is, `import/export` throw errors since they are not recognized.